app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class EmployeesController < ApplicationController
+  before_action :authenticate_employee!
+  before_action :check_employee, only: %i[edit]
+
+  def show
+    render :show, locals: { employee: current_employee }
+  end
+
+  def edit
+    render :edit, locals: { employee: current_employee }
+  end
+
+  def update
+    params[:employee].compact_blank! if params[:employee][:email].blank? && params[:employee][:password].blank?
```

Comment:
mutating params hash is a bad practice. I'd consider creating a method called `update_employee_params` and here perform these operations using maybe `.tap` method

Reply:
I also used policy to modify params